src/groups/translation_group.jl

```diff
@@ -0,0 +1,23 @@
+"""
+    TranslationGroup{𝔽,T}
+
+The Lie group consisting of the [`AdditionGroupOperation`](@ref) on some
+[`Euclidean`](@extref `Manifolds.Euclidean`) space.
+
+# Constructor
+    TranslationGroup(n₁,...,nᵢ; kwargs...)
+
+Generate the translation group on ``𝔽^{n₁,…,nᵢ}`` = `Euclidean(n₁,...,nᵢ; field=𝔽)`,
+which is isomorphic to the group itself. all keyword arguments in `kwargs...`
+are passed on to [`Euclidean`](@extref `Manifolds.Euclidean`) as well
+"""
+const TranslationGroup{𝔽,T} = LieGroup{𝔽,AdditionGroupOperation,Manifolds.Euclidean{T,𝔽}}
+
+function TranslationGroup(n::Int...; kwargs...)
+    Rn = Manifolds.Euclidean(n...; kwargs...)
+    return TranslationGroup{typeof(Rn).parameters[[2, 1]]...}(Rn, AdditionGroupOperation())
+end
+
+function Base.show(io::IO, G::TranslationGroup)
+    return print(io, "TranslationGroup($(G.manifold), $(G.op))")
```

Comment:
I guess `G.op` shouldn't be necessary, and we should have a matching constructor from `G.manifold`??

Reply:
well, spotted, we indeed have one that just requires the dimensions of the manifold even. Will check how to update this.

Reply:
Was even easy to fix. Thanks for spotting, the show is now much nicer with `TranslationGroup(3; field=...)`